src/snowcli/cli/constants.py

```diff
@@ -6,7 +6,7 @@
 
 
 class ObjectType(Enum):
-    COMPUTE_POOL = "compute pool"
+    COMPUTE_POOL = "compute-pool"
```

Comment:
`snow object show compute pool` had no chance to work so I changed it to `snow object show compute-pool`

Reply:
IMO we should consider in the future to do some explicit mapping: `object-type -> (cmd-name, sql-name, sql-plural-name)`. A bit more manual work but IMO it would be much safer.